CONTRIBUTING.md

```diff
@@ -202,3 +204,10 @@ New dependency: dependency_name
 license: SPDX license tag
 origin: https://dependency_origin.com
 ```
+
+## Debugging
+
+### Logging
+
```

Comment:
I got to say, this looks a little weird - to have this one section in web and the other here, in repo...
I guess we should pick one (UR has only Web version). I'd prefer the repo version, since it's always at hand... any opinions...?

Reply:
I'm not sure about BUILD.md but I also think we shouldn't add info about logging to CONTRIBUTE. I think it should be added to the README. If we find that README is too long, then we could probably remove other sections from it.

Reply:
I think I agree with Rafał, re-done, please re-review 😉